heron/api/src/scala/BUILD

```diff
@@ -0,0 +1,52 @@
+licenses(["notice"])
```

Comment:
**General BUILD Comment**:
`*.scala` files are not referenced here. Bazel supports Scala via 
https://github.com/bazelbuild/rules_scala 
so i think we need to use scala_library to compile scala files by referencing
`com/twitter/heron/streamlet/scala/**/*.scala` as `src`.